static/app/components/badge/groupPriority.tsx

```diff
@@ -99,14 +99,8 @@ export function makeGroupPriorityDropdownOptions({
 }
 
 export function GroupPriorityBadge({priority, children}: GroupPriorityBadgeProps) {
-  const organization = useOrganization();
   return (
-    <StyledTag
-      type={getTagTypeForPriority(priority)}
-      isBigtag={organization.features.includes(
-        'issue-stream-new-assignee-dropdown-trigger'
-      )}
-    >
+    <StyledTag type={getTagTypeForPriority(priority)} isBigtag>
```

Comment:
Since this is the only use of `StyledTag` you can remove the `isBigtag` prop and just make the `height` static

Reply:
Good point! Removing now